modules/interaction/src/blaze/interaction/history/instance.clj

```diff
@@ -72,6 +72,11 @@
   (fn [{::reitit/keys [router match] :keys [query-params]
         {{:fhir.resource/keys [type]} :data} ::reitit/match
         {:keys [id]} :path-params}]
+    (log/debug
+      (if query-params
+        (format "Get [base]/%s/%s/_history?%s" type id query-params)
```

Comment:
Get needs to be uppercase.  query-params need to be formatted in the usual URL style: 
```
k1=v1&k2=v2
```
You can use `clojure.string/join` here.

Reply:
Done!